cfn-resources/search-index/docs/README.md

```diff
@@ -158,6 +162,16 @@ _Type_: String
 
 _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
 
+#### Type
+
+Type of index: **search** or **vectorSearch**. Default type is **search**.
+
+_Required_: No
+
+_Type_: String
+
+_Update requires_: [Replacement](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-replacement)
```

Comment:
Is this intentional? Dont believe we would need to recreate the resource if the index type is changed.

Reply:
This is a consequence of having type as createOnlyProperties. Type can't be changed once it's created.

for example trying to change the type from the Atlas UI will show this error: 
```
type" property for Vector search index must be: "vectorSearch".
```